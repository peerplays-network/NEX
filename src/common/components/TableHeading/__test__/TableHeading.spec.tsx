import { render } from "@testing-library/react";
import React from "react";

import { TableHeading } from "../index";

describe("TableHeading component", () => {
  describe("TableHeading in vote table", () => {
    it("renders correctly for Vote table rank column", () => {
      const dummyComponentProps = {
        heading: "rank",
      };
      const { asFragment } = render(<TableHeading {...dummyComponentProps} />);
      expect(asFragment()).toMatchSnapshot();
    });

    it("renders correctly for Vote table Name column", () => {
      const dummyComponentProps = {
        heading: "name",
      };
      const { asFragment } = render(<TableHeading {...dummyComponentProps} />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
