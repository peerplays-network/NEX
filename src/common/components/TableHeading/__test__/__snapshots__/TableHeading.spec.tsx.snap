// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHeading component TableHeading in vote table renders correctly for Vote table Name column 1`] = `
<DocumentFragment>
  Name
</DocumentFragment>
`;

exports[`TableHeading component TableHeading in vote table renders correctly for Vote table rank column 1`] = `
<DocumentFragment>
  Rank
</DocumentFragment>
`;
